[Runtime] Add FrankenPhpWorkerResponseRunner for simple response return (#63661)
symfony/symfony: src/Symfony/Component/Runtime/CHANGELOG.md

---

 * Add `resolveType()` for customizing how types are resolved in runtimes extending `SymfonyRuntime`

>  * Update `FrankenPhpWorkerRunner` to support applications that return a `Response` object

Reviewer: ```suggestion
 * Add support for apps returning a `Response` object in `FrankenPhpWorkerRunner`
```
